src/components/JobCard/JobCard.tsx

```diff
@@ -43,8 +46,20 @@ const JobCard = ({
   liked
 }: JobCardProps) => {
   const [modalVisible, setModalVisible] = useState(false);
+  const [likeValue, setLikeValue] = useState(liked);
   const visibleMap = objectToMap(visible);
 
+  const toggleLike = () => {
+    setLikeValue((likeValue) => !likeValue);
+  };
+
+  useEffect(() => {
+    const updateFirebase = async () => {
```

Comment:
probably don't need to define another function (updateFirebase) in this useEffect. can just call updateJob